src/common/java/net/minecraftforge/gradle/common/util/MinecraftExtension.java

```diff
@@ -127,4 +132,9 @@ public void accessTransformer(Object file) {
     }
 
     public abstract ConfigurableFileCollection getSideAnnotationStrippers();
+
+    public abstract Property<Boolean> getEnableEclipsePrepareRuns();
+    public abstract Property<Boolean> getEnableIdeaPrepareRuns();
```

Comment:
Might be better to document these briefly, since the name doesn't imply much.

Reply:
I will do so on the [wiki](https://github.com/MinecraftForge/ForgeGradle/wiki/Configuration#properties) if / after the PR is merged,

Reply:
Both places would be good.